packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -759,6 +773,35 @@ abstract class SliderTickMarkShape {
     bool isEnabled,
     TextDirection textDirection,
   });
+
+  /// Special instance of [SliderTickMarkShape] to skip the thumb painting.
+  static SliderTickMarkShape noTickMark = EmptySliderTickMarkShape();
+}
+
+/// A special version of [SliderTickMarkShape] that has a zero size and paints
+/// nothing.
+///
+/// This class is used to create a special instance of a [SliderTickMarkShape]
+/// that can be specified in order to skip over, or paint nothing for, a
+/// a tick mark shape.
```

Comment:
This doc should refer to [SliderTickMarkShape.noTickMark]. 

I don't think this class needs to be public.

Reply:
Updated the wording here and in _EmptySliderComponentShape